### PR TITLE
RC2 update for Migrate legacy MVC to Core

### PR DESCRIPTION
**Pull request by Rick-Anderson**

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@ardalis @danroth27 please review - this is an important topic for RC2

**Comment by danroth27** [comment]
@rachelappel Please review as well

File: aspnet/migration/mvc.rst [hunk 1]
```diff
@@ -131,18 +135,20 @@ Upon saving the file, Bower will automatically download the dependencies to the
 
 .. image:: mvc/_static/search.png
 
+.. Note:: *bower.json* is not visible in **Solution Explorer**.
```
Comment (danroth27) [inline aspnet/migration/mvc.rst]: Provide instructions on how to see bower.json via Show All Files option.

File: aspnet/migration/mvc.rst [hunk 2]
```diff
@@ -219,12 +226,40 @@ The updated _Layout.cshtml file is shown below:
...
+
 Configure Bundling
 ------------------
 
 The ASP.NET MVC starter web template utilized the ASP.NET Web Optimization for bundling. In ASP.NET Core, this functionality is performed as part of the build process using `Gulp <http://gulpjs.com/>`__. We've previously configured bundling and minification; all that's left is to change the references to Bootstrap, jQuery and other assets to use the bundled and minified versions. You can see how this is done in the layout file (*Views/Shared/_Layout.cshtml*) of the full template project. See :doc:`/client-side/bundling-and-minification` for more information.
 
+Solving HTTP 500 errors
+-------------------------
+
+There are many problems that can cause a HTTP 500 error message that contain no information on the source of the problem. For example, if the *Views/_ViewImports.cshtml* file contains a namespace that doesn't exist in your project, you'll get a HTTP 500 error. To get a detailed error message, add the following code:
```
Comment (danroth27) [inline aspnet/migration/mvc.rst]: Provide a link to the [Error Handling](https://docs.asp.net/en/latest/fundamentals/error-handling.html) article for more details.

File: aspnet/migration/mvc/samples/WebApp1/src/WebApp1/bower.json [hunk 3]
```diff
@@ -1,8 +1,8 @@
 {
-    "name": "ASP.NET",
-    "private": true,
-    "dependencies": {
-        "bootstrap": "3.3.5",
-        "jquery": "2.1.4"
-    }
-}
+	"name": "asp.net",
```
Comment (danroth27) [inline aspnet/migration/mvc/samples/WebApp1/src/WebApp1/bower.json]: Fix whitespace issue

File: aspnet/migration/mvc/samples/WebApp1/src/WebApp1/project.json [hunk 4]
```diff
@@ -1,34 +1,50 @@
...
-        "node_modules"
-    ],
-    "publishExclude": [
-        "**.user",
-        "**.vspscc"
-    ],
-    "scripts": {
-        "prepublish": [ "npm install", "bower install", "gulp clean", "gulp min" ]
-    }
+  "runtimeOptions": {
```
Comment (danroth27) [inline aspnet/migration/mvc/samples/WebApp1/src/WebApp1/project.json]: Fix to correctly setup serverGC:

```json
  "runtimeOptions": {
    "configProperties": {
      "System.GC.Server": true
    }
  }
```

**Comment by danroth27** [comment]
Some minor comments - :shipit:

**Pull request closed without merging**